core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -148,6 +173,79 @@ oppia.directive('tutorCard', [
             });
           };
 
+          $scope.playPauseAudioTranslation = function() {
+            // TODO(tjiang11): Change from on-demand loading to pre-loading.
+
+            if (cardIndexOfPlayingAudio === undefined) {
+              cardIndexOfPlayingAudio =
+                playerPositionService.getActiveCardIndex();
+
+              // TODO(tjiang11): On first play, ask learner to pick language
+              // and subsequently for confirmation to use bandwidth 
+              // to download audio files.
+            }
+
+            $scope.showExtraAudioControls = true;
+
+            if (!AudioPlayerService.isPlaying()) {
+              if (AudioPlayerService.trackLoaded() && isSameAudio()) {
+                AudioPlayerService.play();
+              } else {
+                loadAndPlayAudioTranslation();
+              }
+            } else {
+              AudioPlayerService.pause();
+              if (!isSameAudio()) {
+                loadAndPlayAudioTranslation();
```

Comment:
I did consider doing that at first, but for some reason thought it'd be nice to listen to audio while looking back at previous cards. But yeah I guess now that you mention it doesn't make too much sense. Will look into doing this.